locale/en/blog/vulnerability/december-2017-security-releases.md

```diff
@@ -1,20 +1,71 @@
 ---
-date: 2017-12-04T19:30:00.617Z
+date: 2017-12-08T16:30:00.617Z
 category: vulnerability
 title: Data Confidentiality/Integrity Vulnerability, December 2017
 slug: december-2017-security-releases
 layout: blog-post.hbs
 author: Michael Dawson
 ---
 
-# Summary
+# _(Update 7-December-2017)_ Security releases available
+
+## Summary
+
+Updates are now available for all active Node.js release lines.  These include the fix for the vulnerability identified in the initial announcement.
+
+In addition the updates for 8.X and 9.X include a fix for a low severity buffer vulnerability as describe below.
+
+We recommend that all users upgrade as soon as possible.
+
+## Downloads
+* [Node.js 9 (Current)](https://nodejs.org/en/blog/release/v9.2.1)
+* [Node.js 8 (LTS "Carbon")](https://nodejs.org/en/blog/release/v8.9.3)
+* [Node.js 6 (LTS "Boron")](https://nodejs.org/en/blog/release/v6.12.2)
+* [Node.js 4 (LTS "Argon")](https://nodejs.org/en/blog/release/v4.8.7)
+
+**Note**: The build for ARMv6 for v8.9.3 is not yet available and will be released later today.
+
+## Data Confidentiality/Integrity Vulnerability - CVE-2017-15896
+
+Node.js was affected by OpenSSL vulnerability CVE-2017-3737 in regards to the use of SSL_read() due to TLS handshake failure. The result was that an active network attacker could send application data to Node.js using the TLS or HTTP2 modules in a way that bypassed TLS authentication and encryption.  
+
+* The original HTTP module was not affected.
+
+* The vulnerability in the HTTP2 module (which only existing in the 8.X and 9.X lines) was fixed through nodejs/node@f3686f2. HTTP2 was previously exploitable through the submission of malicious data by an attacker.
```

Comment:
Do you want to keep the revision as `nodejs/node@f3686f2` (or change to link to GitHub?)